pyproject.toml

```diff
@@ -0,0 +1,21 @@
+[build-system]
+requires = ["setuptools"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "benevolent"
+version = "0.0"
+description = "a secret codes image processing tool"
+requires-python = ">=3.9"
+readme = "README.md"
+
+dynamic = ["dependencies"]
+
+[project.scripts]
+benevolens = "benevolent.__main__:cli"
+
+[tool.setuptools]
+packages = ["benevolent"]
+
+[tool.setuptools.dynamic]
+dependencies = {file = ["requirements.txt"]}
```

Comment:
i'm quite happy with this.  in the past, i had duplicated `requirements.txt` list in `pyproject.toml`.  turns out that's not necessary